### PR TITLE
Fixes #5. No tests added yet

### LINKED ISSUE #5: Pin/unpin polls automatically
Pin is simple, at /newevent.
Unpin, unclear but after its completetion (however that's defined)

### PR DESCRIPTION
**Pull request by walkerjens**

### CODE REVIEW

**Comment by walkerjens** [comment]
Also fixes #6 in addition to #5

**Review by tingvarsson — CHANGES_REQUESTED** [review]

File: ongabot/handler/canceleventcommand.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    pinned_poll = context.chat_data.get("pinned_poll_msg")
+
+    if pinned_poll is not None:
+        pinned_poll.unpin()
+        context.chat_data["pinned_poll_msg"] = None
+
+        context.bot.send_message(
+            update.effective_chat.id,
+            "Event for "
+            + get_upcoming_wednesday_date(date.today()).strftime("%Y-%m-%d")
```
[the remark this answers is not in the document]
Comment (walkerjens) [reply ongabot/handler/canceleventcommand.py:31]: Changed message to contain date from poll.

File: ongabot/handler/neweventcommand.py [hunk 2]
```diff
@@ -20,6 +20,22 @@ def callback(update: Update, context: CallbackContext):
     logger.debug("update:")
     logger.debug("%s", update)
 
+    # Retrieve prev pinned msg and unpin
+    pinned_poll = context.chat_data.get("pinned_poll_msg")
+
+    if pinned_poll is not None:
+        next_wed = get_upcoming_wednesday_date(date.today())
+        if next_wed.strftime("%Y-%m-%d") in pinned_poll.poll.question:
```
[the remark this answers is not in the document]
Comment (walkerjens) [reply ongabot/handler/neweventcommand.py:28]: We still need to match the date stored with upcoming wednesday date. In this manner we keep the original data structure (message) intact which obfuscates less than a custom data structure. I vote that this way remains, at least until any further benefit from a custom struct is evident.

[the remark this answers is not in the document]
Comment (tingvarsson) [reply ongabot/handler/neweventcommand.py]: aight.

**Review by tingvarsson — APPROVED** [review]

**Pull request merged**